conda/environments/cuml_dev_cuda10.0.yml

```diff
@@ -10,15 +10,17 @@ dependencies:
 - cmake=3.14.5
 - numba=0.44*
 # - cupy>=6*
-- cudf=0.8*
-- rmm=0.8*
+- cudf=0.10*
+- rmm=0.10*
 - cython>=0.29,<0.30
 - pytest>=4.6
-- scikit-learn=0.21
+- scikit-learn>=0.21
 - umap-learn>=0.3.9
 - scikit-learn>=0.21
 - dask
 - distributed
 - dask-ml
-- dask-cuda=0.8*
-- dask-cudf=0.8*
+- dask-cuda=0.10*
+- dask-cudf=0.10*
+- nccl>=2.4
```

Comment:
I might do `>=2.4,<3` so we don't get a crazy unexpected upgrade sometime, especially if we're using this in the future e.g. to repro a benchmark or something.